docs/sql-migration-guide.md

```diff
@@ -34,6 +34,8 @@ license: |
 
   - In Spark 3.1, structs and maps are wrapped by the `{}` brackets in casting them to strings. For instance, the `show()` action and the `CAST` expression use such brackets. In Spark 3.0 and earlier, the `[]` brackets are used for the same purpose. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.castComplexTypesToString.enabled` to `true`.
 
+  - In Spark 3.1, `CAST` converts NULL elements of structures, arrays and maps to "null". In Spark 3.0 or earlier, NULL elements are converted to empty strings. To restore the behavior before Spark 3.1, you can set `spark.sql.legacy.castComplexTypesToString.enabled` to `true`.
```

Comment:
`In Spark 3.1, NULL elements of structures, arrays and maps are converted to "null" in casting them to strings.`

Reply:
The passive stuff was not recommended by IntelliJ IDEA ;-) Native speakers don't like it.

Reply:
<img width="881" alt="Screenshot 2020-08-05 at 09 34 31" src="https://user-images.githubusercontent.com/1580697/89379682-e594e880-d6fe-11ea-8594-040a428cdce3.png">

Reply:
OK, but let's highlight it's for "casting to string", not for general `CAST`

Reply:
I have replaced it by your sentence already.